Sources/MintKit/Mint.swift

```diff
@@ -52,20 +52,48 @@ public struct Mint {
         return metadata.packages.first(where: { $0.key.lowercased().contains(name.lowercased()) })?.key
     }
 
+    func getGlobalInstalledPackages() -> [String: String] {
+        guard installationPath.exists,
+            let packages = try? installationPath.children() else {
+                print("No mint packages installed to global")
+                return [:]
+        }
+
+        return packages.reduce(into: [:]) { result, package in
+            guard let installStatus = try? InstallStatus(path: package, mintPackagesPath: path),
+                case .mint = installStatus.status,
+                let symlink = try? package.symlinkDestination() else {
+                    return
+            }
+
+            var componets = symlink.components
+            componets.removeLast()
+            let version = componets.removeLast()
```

Comment:
The version is already an associated type (`InstallStatus.mint(version: String)`) so you can just use that 👍